src/components/PlantingSites/PlantingSitesList.tsx

```diff
@@ -52,13 +72,13 @@ export default function PlantingSitesList(): JSX.Element {
       setPlantingSites(transformedResults);
     }
     setSearchResults(transformedResults);
-  }, [selectedOrganization, debouncedSearchTerm, searchSortOrder, timeZones, defaultTimeZone]);
+  }, [debouncedSearchTerm, filters.projectIds, selectedOrganization.id, searchSortOrder, timeZones, defaultTimeZone]);
 
   useEffect(() => {
     onSearch();
   }, [selectedOrganization, onSearch]);
 
-  if (plantingSites && !plantingSites.length) {
+  if (plantingSites && !filtersEmpty && !plantingSites.length) {
```

Comment:
should this be `&& filtersEmpty`

Reply:
No because if the filters are empty, and there are no planting sites, we do want to show the empty state page. If there are filters, but no planting sites, then we should show an empty table (assumption)

Reply:
That is correct, here we are returning the empty state page if the filters are not empty and planting sites are empty. Hence my comment..

Reply:
Ah, yea, you're right. It was working locally because I didn't actually call the function 🤦‍♂️